ride_with_me_backend/rides/schema.py

```diff
@@ -0,0 +1,25 @@
+import graphene
+from graphene_django import DjangoObjectType
+from django.db.models.functions import Concat
+from django.db.models import Value
+
+from .models import City
+
+
+class CityType(DjangoObjectType):
+    name = graphene.String()
```

Comment:
This is to have a custom field for City, 'name'. Only works on the graphql endpoint though.